README.md

```diff
@@ -766,6 +767,25 @@ UI available. Selectrum also covers the most important aspects of the
 API and strives to achieve full compliance, as well. For the few edge
 cases left, see the Caveats section.
 
+### Vertico
+
+[Vertico](https://github.com/minad/vertico) is a new minimalistic
+completion system based on the Emacs default completion offering a
+similar UI as Selectrum. It uses a different implementation approach -
+it extends the default completion system in a similar way as Icomplete
+and and is therefore fully compliant with all features of the
+`completing-read` API. Overall Vertico follows a similar philosophy as
+Selectrum, relying on default components and [complementary
+packages](#complementary-extensions). Many of the complementary
+packages work well with both Selectrum and Vertico. However Selectrum
+and Vertico differ in their priorities. Selectrum focuses on offering
+the best possible UI, e.g., supports both a horizontal and a vertical
```

Comment:
Maybe mention caching here? I'm not sure about focus on best possible UI don't  all UIs try to do this? At least they should ;) Maybe something like "flexible UI which tries to improve on default completion in some cases"?